exporting/pubsub/pubsub.c

```diff
@@ -56,6 +56,35 @@ int init_pubsub_instance(struct instance *instance)
     return 0;
 }
 
+/**
+ * Clean a PubSub connector instance
+ *
+ * @param instance an instance data structure.
+ */
+void clean_pubsub_instance(struct instance *instance)
+{
+    info("EXPORTING: cleaning up instance %s ...", instance->config.name);
+
+    struct pubsub_specific_data *connector_specific_data =
+        (struct pubsub_specific_data *)instance->connector_specific_data;
+    pubsub_cleanup(connector_specific_data);
```

Comment:
Should we test `connector_specific_data` before to clean?

Reply:
There are so many places where we don't check the pointer. It's pointless to check it on exit.

Reply:
I was thinking in the possibility that another contributor could forget to initialize them in the future and we were taking care to avoid clean what was never allocated.